modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/js/asset_categories_selector/AssetCategoriesFriendlyUrlSelector.js

```diff
@@ -101,16 +103,21 @@ function AssetVocabulariesCategoriesFriendlyUrlSelector({
 		[selectedItems]
 	);
 
-	useEffect(() => {
-		const titleInput = document.getElementById(namespace + 'title');
-
-		if (titleInput && customUrlCheckboxValue !== 0) {
-			titleInput.addEventListener('change', (event) => {
+	useEventListener(
+		'change',
+		(event) => {
+			if (!editedRef.current) {
 				setFriendlyUrlValue(normalizeFriendlyURL(event.target.value));
-			});
-			setFriendlyUrlValue(normalizeFriendlyURL(titleInput.value));
-		}
-	}, [customUrlCheckboxValue, namespace]);
+			}
+		},
+		true,
+		document.getElementById(namespace + 'title')
+	);
+
+	const handleChange = (event) => {
+		editedRef.current = true;
+		setFriendlyUrlValue(normalizeFriendlyURL(event.target.value));
+	};
```

Comment:
I created the editedRef, so we keep updating the placeholder value to match the current title, until the user actually edits the field. After that point we are no longer syncing the two fields.

Reply:
Okay, I get it now. 

But I'm not sure if it's accurate. 

When a user edits the post (after it is saved) this mechanism will fail, because we consider it **unedited** always on load.

Maybe it is better to check customUrlCheckboxValue like before, and only update it when it is DEFAULT_URL.